tests/php/classes/test-class-wc-rest-connect-shipping-carrier-types-controller.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+
+
+/**
+ * Unit test for WC_Connect_API_Client_Live
+ */
+class WP_Test_WC_REST_Connect_Shipping_Carrier_Types_Controller extends WC_Unit_Test_Case {
+
+	/** @var WC_Connect_API_Client_Live $api_client_mock */
+	protected $api_client_mock;
+
+	/** @var WC_Connect_Service_Settings_Store $setting_store_mock */
+	protected $setting_store_mock;
+
+	/** @var WC_Connect_Logger $connect_logger_mock */
+	protected $connect_logger_mock;
+
+	/**
+	 * @inherit
+	 */
+	public static function setupBeforeClass() {
+		require_once dirname( __FILE__ ) . '/../../../classes/class-wc-connect-api-client-live.php';
+		require_once dirname( __FILE__ ) . '/../../../classes/class-wc-connect-service-settings-store.php';
+		require_once dirname( __FILE__ ) . '/../../../classes/class-wc-connect-logger.php';
+		require_once dirname( __FILE__ ) . '/../../../classes/class-wc-rest-connect-base-controller.php';
+		require_once dirname( __FILE__ ) . '/../../../classes/class-wc-rest-connect-shipping-carrier-types-controller.php';
```

Comment:
These files are imported this way because our unit test bootstrap doesn't autoload all files for the plugin. I think we can improve this either by 
1. Create a test bootstrap for all these files, for all tests in WCS
2. Create an autoloader for WCS to register all WCS scripts.

But probably not this PR, something in the future.